packages/synchro-charts/src/utils/dataTypes.ts

```diff
@@ -81,6 +81,15 @@ export interface SizeConfig extends MinimalSizeConfig {
   marginBottom: number;
 }
 
+export interface DialSizeConfig extends MinimalSizeConfig {
+  fontSize: number;
+  dialThickness: number;
+  iconSize: number;
+  labelSize: number;
+  unitSize: number;
+  viewport: number;
```

Comment:
doesn't need viewport here

Reply:
The viewport is for box width, like `a 75px by 75x box` , viewport = 75.
Is that  the wrong name `viewport`?

Reply:
we have a very specific concept of `viewport` in synchrocharts, which is `{ yMin, yMax, start: Date, end: Date }` and not a word we should overload. maybe we just call it `width`?

Reply:
Done --- Update `viewport` to `width`